src/App.jsx

```diff
@@ -13,46 +15,112 @@ export const goodsFromServer = [
   'Jam',
   'Garlic',
 ];
+const SORT_FIELD_ALPHABETICALLY = 'alphabetically';
+const SORT_FIELD_LENGTH = 'length';
```

Comment:
it will be better to combine such kind sort types in common object
because  theoretically, the number of types can increase
It is also more convenient to keep constants in a separate file
```suggestion
export const SORT_TYPE = {
  ALPHABETICALLY: 'alphabetically',
  LENGTH: 'length',
};

```